__tests__/Users.js

```diff
@@ -0,0 +1,237 @@
+import '@testing-library/jest-dom/extend-expect';
+
+import { fireEvent, wait, within } from '@testing-library/react';
+import { forceRemoveAll } from 'lib/flashMessage';
+import nock from 'nock';
+import { StatusCodes } from 'shared';
+import { UsersRoutes } from 'shared/constants/routes';
+import {
+  API_ENDPOINT,
+  appCatalogsResponse,
+  AWSInfoResponse,
+  getMockCall,
+  gsOrgResponse,
+  invitesResponse,
+  ORGANIZATION,
+  orgResponse,
+  orgsWithGSResponse,
+  USER_EMAIL,
+  userResponse,
+  usersResponse,
+  V4_CLUSTER,
+  v4AWSClusterResponse,
+  v4AWSClusterStatusResponse,
+  v4ClustersResponse,
+} from 'testUtils/mockHttpCalls';
+import { renderRouteWithStore } from 'testUtils/renderUtils';
+
+describe('Users', () => {
+  beforeAll(() => {
+    nock.disableNetConnect();
+  });
+
+  afterAll(() => {
+    nock.enableNetConnect();
+  });
+
```

Comment:
I like this approach of placing the mock call inside a `describe` so we can have different calls defined for each test suite, but ¿Is there a reason for placing `nock.disableNetConnect()` and `nock.enaableNetConnect()` here? Don't you think we want to have these at top level or even in a global a config file?

Reply:
The `describe` block doesn't do anything functionality wise. Jest isolates every file from the others automatically. I only use `describe` blocks just because they give a better output while running tests.

I added `nock.disableNetConnect();` and `nock.enableNetConnect();` because they give us consistent results on both our local machines and in CI. Having them in a global config is a good idea, but I believe we should move them after we finish refactoring the old tests, since they will probably break some of the tests.

We already had them in a couple of tests, so I didn't introduce them now:
![image](https://user-images.githubusercontent.com/13508038/75329597-22230500-5880-11ea-8262-95126507d6d2.png)

Reply:
Yeah, we can do it after we refactor all tests 👍 

> The `describe` block doesn't do anything functionality wise. Jest isolates every file from the others automatically. I only use `describe` blocks just because they give a better output while running tests.

I thought that it made a difference, check it out: https://jest-bot.github.io/jest/docs/api.html#aftereachfn

Reply:
Sorry, I meant that in this case the describe block doesn't do anything, because it wraps all the tests in the file. So having the `beforeAll` call outside the `describe` block would be the same thing as having it inside it